src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -2,159 +2,257 @@
+
+## Check the status of support tickets [#support]
+
+Ask New Relic AI for a summary of your support tickets to reduce the time to resolution. Revisit solutions and past issues to streamline management. You can also check to see if tickets remain open or can be closed as resolved. 
 
-You can take your investigation further by telling our AI about your stack and asking follow up questions. For example, you can say: 
+Chat with AI and ask:
 
 ```
-Hi, I have an Azure environment and I am having issues with Redis being periodically unavailable. Can you tell me more about this error? How can I diagnose this issue? 
+Summarize my support ticket(s).
+```
+```
+What is the status of my support ticket(s)?
+```
+
+## Demystify your dashboards [#dashboards]
+
+Dashboards are an incredibly useful tool but if you're investigating telemetry on a dashboard you don't own, you might not understand all the nuances. New Relic AI can be your personal data analyst, automatically summarizing key information and uncovering hidden insights directly within the New Relic UI. 
+
+    1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DNT> > <DNT>**Dashboards**</DNT>
+    2. Click **Ask AI** in the top right-hand corner.
+
+Even basic questions can provide detailed summaries that can help clarify complicated data. Try asking: 
+
+```
+What is on this dashboard?
+```
 
 ```
+Summarize this dashboard for me. 
+```
+
+## Accuracy and security [#secure-data]
+
+Our AI assistant uses a variety of inputs to improve its responses and tries to provide the best suggestion to your prompt. For example, it uses inputs like our product documentation, an extensive library of example queries, and your telemetry data. 
+
+For more information about New Relic's security controls, please see our [Security Policy](/docs/licenses/license-information/referenced-policies/security-policy/).
+
+
+<Callout variant="important">
+New Relic AI is powered by Generative AI and is subject to Generative AI's [Service Specific Terms](https://newrelic.com/termsandconditions/service-specific). 
+</Callout>
+
+## Troubleshooting [#troubleshoot]
+
+If you encounter one of these errors while using New Relic AI, refer to the following troubleshooting steps:
+
+**Your user role does not have access to New Relic AI in this account. Please try a different account or contact your New Relic admin.**
+    * Problem: The New Relic account you are currently using does not have New Relic AI enabled, or your user role within the account does not have permission to access New Relic AI.
+    * Solution: Try accessing New Relic AI from a different New Relic account that has the feature enabled. If you believe you should have access, contact your New Relic account administrator to request that they enable New Relic AI for your account or adjust your user role permissions.
+
+**NRAI is not currently available for HIPAA or FedRAMP accounts. Please try a different account.**
```

Comment:
This statement of the issue includes the fix you mention below. Consider a shorter intro like this: HIPAA and Fedramp.

Reply:
See above.